templates/email.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Let's create a `base.html` which includes `<head>`, `<style>`, footer etc and use it for all emails.

Reply:
This html file should contain only context related parts.